src/bundle/Resources/public/js/scripts/fieldType/base/base-preview-field.js

```diff
@@ -270,6 +270,8 @@
             window.addEventListener('drop', this.preventDefaultAction, false);
             window.addEventListener('dragover', this.preventDefaultAction, false);
 
+            this.fieldContainer.addEventListener('ez-show-preview', () => this.showPreview());
```

Comment:
I think `ez-show-preview` is too generic, I would make it more like: `ez-field-type-show-preview` or even give a possibility to fire custom event per "final" class not generic one for all use cases.